src/components/m-policy-feature-item/index.scss

```diff
@@ -0,0 +1,56 @@
+@import "../../styles/settings/colors";
+@import "../../styles/typo/typo-mixins";
+@import "../../styles/mixins/word-break";
+
+$icon-height-sm: 42px;
```

Comment:
Try naming the variables like this: `$policy-feature-item-icon-height-sm` or `$policy-feature-item-icon-width-sm`.

Reply:
Done.